File: examples/sharing/invite-create.swift
import JsBaoClient

// Invite a new user into the app by email (admin/owner, or member when
// member-invitations are enabled).
func inviteUser(client: JsBaoClient) async throws {
  // #region example
  _ = try await client.invitations.create(params: [
    "email": "alice@example.com",
    "role": "member",
  ])
  // #endregion example
}

File: examples/sharing/invite-create.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Invite a new user into the app by email (admin/owner, or member when
// member-invitations are enabled).
export async function inviteUser(client: JsBaoClient) {
  // #region example
  await client.invitations.create({
    email: "alice@example.com",
    role: "member",
  });
  // #endregion example
}
